### DISCUSSION TITLE
galileo test net 不支持用 EthStorage 方式上传文件？

### DISCUSSION THREAD

**Discussion by linghuccc** [IntensiveCoLearning/Web3-URL#170] [Q&A] [answered]
![e](https://github.com/user-attachments/assets/0ce68240-4e04-4495-8677-c788d8613273)

**Comment by qzhodl** [comment, marked as answer]
galileo不支持eip4844 BLOB，所以只能使用 `-t 1`。或者可以EthStorage最新的测试网：https://github.com/ethstorage/pm/blob/main/L2/testnet_usage.md 来部署，可以支持 `-t 2`
